srai/loaders/_base.py

```diff
@@ -11,13 +10,13 @@ class Loader(abc.ABC):
     """Abstract class for loaders."""
 
     @abc.abstractmethod
-    def load(self, area: Union[gpd.GeoDataFrame, Path]) -> gpd.GeoDataFrame:  # pragma: no cover
+    def load(self, *args: Any, **kwargs: Any) -> gpd.GeoDataFrame:  # pragma: no cover
```

Comment:
I'm against args and kwargs. Are we able to make the descendants access only `area` and get other data via constructor?

Reply:
If we want to have a wide selection of loaders, we won't be able to stick to `area` only. GTFS loader for example, requires a file to load from and does not use `area`. 
Maybe we cannot have a single `Loader` interface, if this is too general to cover all cases?